docs/source/datasets/index.rst

```diff
@@ -121,59 +128,75 @@ The following columns are included in this dataset:
      - Notes
    * - Unique simulant ID
      - :code:`simulant_id`
-     - Not affected by noise functions; intended use is "ground truth" for testing and validation. 
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
    * - Unique household ID
      - :code:`household_id`
      - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
        datasets.
    * - First name
      - :code:`first_name`
-     - 
+     -
    * - Middle initial
      - :code:`middle_initial`
-     - 
+     -
    * - Last name
      - :code:`last_name`
-     - 
+     -
    * - Age
-     - :code:`age`  
+     - :code:`age`
      - Rounded down to an integer.
    * - Date of birth
      - :code:`date_of_birth`
      - Formatted as YYYY-MM-DD.
    * - Physical address street number
      - :code:`street_number`
-     - 
+     -
    * - Physical address street name
      - :code:`street_name`
-     - 
+     -
    * - Physical address unit number
      - :code:`unit_number`
-     - 
+     -
    * - Physical address city
-     - :code:`city`    
-     - 
+     - :code:`city`
+     -
    * - Physical address state
-     - :code:`state`  
-     - 
+     - :code:`state`
+     -
    * - Physical address ZIP code
      - :code:`zipcode`
-     - 
-   * - Sex 
-     - :code:`sex`  
+     -
+   * - Housing type
+     - :code:`housing_type`
+     - Possible values for housing type are "Household" for an individual
+       household, or one of six different types of group quarters. The types of
+       instiutional group quarters are "Carceral", "Nursing home", and "Other
+       institutional". The types of noninstitutional group quarters are
+       "College", "Military", and "Other noninstitutional".
+   * - Relationship to reference person
+     - :code:`relationship_to_reference_person`
```

Comment:
As discussed [on slack](https://ihme.slack.com/archives/C02KUQ9LX32/p1689804298425519), our data column in the decennial census is called `relation_to_reference_person` instead. Should I change this to match, or should we change the data to match what's in the docs?

Reply:
I'm leaning towards `relationship_to_reference_person` at this moment.